test/unless.tests.js

```diff
@@ -242,22 +243,22 @@ describe('express-unless', function () {
       return req.baba;
     });
 
-    it('should not call the middleware when the custom rule match', function () {
```

Comment:
The async/await here are not necessary, also can you duplicate the tests for the async vs not async case? 

Thank you